### ISSUE TITLE
seg fault with new release 0.215.0

### ISSUE THREAD

**Issue by MahsaAbadian** [planetscale/cli#939]
Scenario:
- I am running pscale on a docker container on MacOS, I could successfully run the command :

`yarn run db --execute "prisma db push"`

However right after this upgrade I am getting the following error:

```
Stack trace: 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x18 pc=0x104e05188]

goroutine 1 [running]:
go.uber.org/zap/exp/zapslog.NewHandler(...)
	go.uber.org/zap/exp@v0.3.0/zapslog/handler.go:59
github.com/planetscale/cli/internal/proxyutil.New({0x14000488000, {0x0, 0x0}, {0x140003140c0, 0x16}, {0x140003140d8, 0x14}, {0x1400013e000, 0x35}})
	github.com/planetscale/cli/internal/proxyutil/psdb.go:23 +0xb8
github.com/planetscale/cli/internal/cmd/connect.ConnectCmd.func1(0x14000406f00?, {0x140000cf1a0, 0x2, 0x104f14532?})
	github.com/planetscale/cli/internal/cmd/connect/connect.go:158 +0x850
github.com/spf13/cobra.(*Command).execute(0x1400040d508, {0x1400003c0a0, 0x6, 0x6})
	github.com/spf13/cobra@v1.8.1/command.go:985 +0x834
github.com/spf13/cobra.(*Command).ExecuteC(0x105b672e0)
	github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	github.com/spf13/cobra@v1.8.1/command.go:1034
github.com/planetscale/cli/internal/cmd.runCmd({0x105301160, 0x140002ccb90}, {0x1050d0b98, 0x7}, {0x1050d0ba0, 0x7}, {0x1050d0cc0, 0x14}, 0x140002825e8, 0x14000282600, ...)
	github.com/planetscale/cli/internal/cmd/root.go:230 +0xe10
github.com/planetscale/cli/internal/cmd.Execute({0x105301160, 0x140002ccb90}, 0x140002d27e0, {0x14000063860, 0x1, 0x1}, {0x1050d0b98, 0x7}, {0x1050d0ba0, 0x7}, ...)
	github.com/planetscale/cli/internal/cmd/root.go:110 +0x244
main.realMain()
	./main.go:40 +0x19c
main.main()
	./main.go:18 +0x1c
```

**Comment by mscoutermarsh** [comment]
Thank you, we're on it.

**Comment by mscoutermarsh** [comment]
This has been fixed in `v0.216.0`, sorry for the trouble & thank you for the quick report.
